src/Assets/AssetFolder.php

```diff
@@ -98,7 +98,7 @@ public function lastModified()
 
     public function save()
     {
-        $this->disk()->makeDirectory($this->path());
+        $this->disk()->put($this->path().'/.gitkeep', '');
```

Comment:
Is this necessary? Was it fixing something in particular?

Reply:
Yes, before empty folders would not get into version control. If you do a checkout (e.g. while deploying to a server with Forge) any folders without assets would be missing. Adding a .gitkeep file prevents that now and makes sure the empty folders show up.